queries.sql

```diff
@@ -7,4 +7,40 @@ SELECT date_of_birth FROM animals WHERE name='Agumon' OR name='Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> ' Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- update tables with transactions
+BEGIN TRANSACTION;
+UPDATE animals
+SET species = 'unspecified';
+ROLLBACK;
```

Comment:
- [ ] Kindly make sure to add query to verify the change was made before ROLLBACK and after ROLLBACK to verify if the change was undone.

```sql
   SELECT species from animals;
```

Reply:
well resolved! ✔️